.eslintrc.js

```diff
@@ -34,27 +34,19 @@ module.exports = {
 
   rules: {
     "babel/new-cap": "off",
-    "jsdoc/check-param-names": "warn",
-    "jsdoc/check-tag-names": "warn",
-    "jsdoc/check-types": "warn",
-    "jsdoc/newline-after-description": "warn",
-    "jsdoc/require-param": "warn",
-    "jsdoc/require-param-description": "warn",
-    "jsdoc/require-param-name": "warn",
-    "jsdoc/require-param-type": "warn",
-    "jsdoc/require-returns-description": "warn",
-    "jsdoc/require-returns-type": "warn",
+    "jsdoc/require-returns-description": "off",
```

Comment:
Turning this rule off lets us have a `@returns {void}` without requiring a description.